lib/iris/tests/test_netcdf.py

```diff
@@ -25,7 +25,6 @@
 # importing anything else.
 import iris.tests as tests
 
-import os
```

Comment:
:unamused: I could have sworn I checked that ... I remember checking *something*!